piplicenses/const.py

```diff
@@ -0,0 +1,128 @@
+from argparse import Namespace
+from enum import Enum, auto
+from typing import List, Optional
+
+try:
+    from prettytable.prettytable import ALL
+    PTABLE = True
+except ImportError:  # pragma: no cover
+    PTABLE = False
+
+
+# General information
+__pkgname__ = 'pip-licenses'
+__version__ = '3.3.0'
+__author__ = 'raimon'
+__license__ = 'MIT'
+__summary__ = ('Dump the software license list of '
+               'Python packages installed with pip.')
+__url__ = 'https://github.com/raimon49/pip-licenses'
```

Comment:
It's a good idea that the constants for the whole package are now in `const.py` 👍 

On the other hand, it seems that meta-information such as `__version__` is often declared in `__init__.py`.

What do you think?

Reply:
I've seen it being declared in `const.py` as well. Personally I like it a bit more that way. It's easier to use use these in the remaining code. Eg. the argparser `-v` command.

Reply:
Okay, let's just let it be declared in `const.py`.